lab_1/main.py

```diff
@@ -25,8 +34,14 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
-
+    if type(tokens) == list:
+        if type(stop_words) == list:
```

Comment:
better move this condition on the level above and use `if not ... : ` to avoid nesting

Reply:
nesting is quite bad, especially if it can be avoided